apps/docs/src/app/documentation/core-helpers/stackblitz/code-example-stack/styles.scss

```diff
@@ -2,47 +2,47 @@
 
 @font-face {
     font-family: '72';
-    src: url('node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_base_fiori/fonts/72-Regular-full.woff')
+    src: url('node_modules/@sap-theming/theming-base-content/content/Base/baseLib/baseTheme/fonts/72-Regular-full.woff')
         format('woff');
     font-weight: normal;
     font-style: normal;
 }
 
 @font-face {
     font-family: '72';
-    src: url('node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_base_fiori/fonts/72-Light.woff')
+    src: url('node_modules/@sap-theming/theming-base-content/content/Base/baseLib/baseTheme/fonts/72-Light.woff')
         format('woff');
     font-weight: 300;
     font-style: normal;
 }
 
 @font-face {
     font-family: '72';
-    src: url('node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_base_fiori/fonts/72-Bold.woff')
+    src: url('node_modules/@sap-theming/theming-base-content/content/Base/baseLib/baseTheme/fonts/72-Bold.woff')
         format('woff');
     font-weight: 700;
     font-style: normal;
 }
 
 @font-face {
     font-family: 'SAP-icons';
-    src: url('node_modules/@sap-theming/theming-base-content/content/Base/baseLib/sap_fiori_3/fonts/SAP-icons.woff')
+    src: url('node_modules/@sap-theming/theming-base-content/content/Base/baseLib/baseTheme/fonts/SAP-icons.woff')
```

Comment:
it didn't need to be changed

Reply:
<img width="347" alt="Screen Shot 2021-06-20 at 9 09 53 AM" src="https://user-images.githubusercontent.com/4380815/122675417-51f9c880-d1a7-11eb-8537-290edfd01301.png">